pkg/karmadactl/addons/addons.go

```diff
@@ -18,7 +18,8 @@ var (
 		
 		1. karmada-descheduler
 		2. karmada-scheduler-estimator
-		3. karmada-search`)
+		3. karmada-search
+		4. karmada-metrics-adapter`)
```

Comment:
how about sorting the addons in alphabetic order

Reply:
Updated.